ui/ui-components/pages/model/[modelId]/source/[sourceId]/schedule.tsx

```diff
@@ -73,10 +73,17 @@ export default function Page(props) {
 
     if (response?.schedule) {
       if (response.schedule.state === "ready" && schedule.state === "draft") {
-        router.push(
-          "/model/[modelId]/source/[sourceId]/overview",
-          `/model/${source.modelId}/source/${source.id}/overview`
-        );
+        if (totalSources === 1 && totalProperties === 1) {
+          router.push(
+            "/model/[modelId]/source/[sourceId]/multipleProperties",
+            `/model/${source.modelId}/source/${source.id}/multipleProperties`
+          );
+        } else {
+          router.push(
+            "/model/[modelId]/source/[sourceId]/overview",
+            `/model/${source.modelId}/source/${source.id}/overview`
+          );
+        }
```

Comment:
A better implementation is to not duplicate the calls but instead duplicate the final page value because we know in either case we'll redirect.